internal/wazeroir/format.go

```diff
@@ -14,193 +11,13 @@ func Format(ops []Operation) string {
 
 	_, _ = buf.WriteString(EntrypointLabel + "\n")
 	for _, op := range ops {
-		formatOperation(buf, op)
-	}
-
-	return buf.String()
-}
-
-func formatOperation(w io.StringWriter, b Operation) {
```

Comment:
this format was inconsistent with OperationKind.String() and clearly was a garbage.